### PR TITLE
documentation/readme-section

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** Internal Ticket

**This PR will result in the following new package version:** N/A

This will not result in a new release at the moment. Instead we will merge this into `main` and the README update will be integrated into the next upcoming release. This edit was only required so the models descriptions may be viewed in the Fivetran UI for Quickstart.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [n/a] dbt run –full-refresh && dbt test
- [n/a] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked, tagged, and properly assigned
- [X] All necessary documentation and version upgrades have been applied
    
- [n/a] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [n/a] Detailed validation steps have been provided below

#### ❗ Special Updates for Ad Reporting ❗
To reduce integration testing time, not all models should be enabled in the `run_models.sh` vars. Update the variables after `dbt run` and `dbt test` to set:
- [n/a] this PR's package to `true`
- [n/a] Google Ads and Facebook Ads to `true` (if not already)
- [n/a] All other packages to `false`

### Detailed Validation
Please share any and all of your validation steps:

No validations were required for this update since this was only a README edit which will have no impact on users of the data model.

### If you had to summarize this PR in an emoji, which would it be?
 
📑

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.NEXT.RELEASE
+
+## Under the Hood
+- Addition of a section tag within the README so the model descriptions may be accessible within the Fivetran UI for Quickstart. ([PR #118](https://github.com/fivetran/dbt_ad_reporting/pull/113))
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:4]:
```suggestion
- Addition of a section tag within the README so the model descriptions may be accessible within the Fivetran UI for Quickstart. ([PR #113](https://github.com/fivetran/dbt_ad_reporting/pull/113))
```

**Review by Fivetran-milanjovic — APPROVED** [review]
Thank you!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.NEXT.RELEASE
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:1]:
```suggestion
# dbt_ad_reporting vNEXTRELEASE
```
My only suggestion is to format the version without the dots so the auto releaser doesn't run.

**Review by fivetran-catfritz — APPROVED** [review]
Small suggestion but otherwise lgtm!

**Pull request merged**